### PR TITLE
Updated Resize error message and added link

### PR DESCRIPTION
**Pull request by Juliawna**
When a customer tries to resize their Linode to a smaller plan without first resizing their disks, they trigger an error message. 

I updated the copy of this error message, and added a link to our "Resize a Linode" document for instructions on how to size down a plan.

### CODE REVIEW

**Comment by Jskobos** [comment]
I worked with Julianna on this, should have been a simple change but there's some TypeScript stuff that makes it weird. I'm not super confident about the approach we used, feedback welcome. Also I'm not sure that I like blue link text inside a red error message.

File: packages/manager/src/features/linodes/LinodesDetail/LinodeResize/LinodeResize_CMR.tsx [hunk 1]
```diff
@@ -90,6 +91,9 @@ const styles = (theme: Theme) =>
     },
     selectPlanPanel: {
       marginTop: theme.spacing(5)
+    },
+    error: {
+      color: '#C44742'
```
Comment (Jskobos) [inline packages/manager/src/features/linodes/LinodesDetail/LinodeResize/LinodeResize_CMR.tsx:96]: We can delete this, I think. It's displayed inside a Notice with a red border, we don't usually color the text inside them as well.

Comment (DevDW) [reply packages/manager/src/features/linodes/LinodesDetail/LinodeResize/LinodeResize_CMR.tsx:96]: Agreed; I do think our error messages have a bolder weight than this currently has though

**Review by DevDW — APPROVED** [review]
Left a comment re: font weight of the error message. Aside from that, setting `error` to some JSX does feel a little clunky, but I can't think of a better approach at the moment that wouldn't be clunkier.

**Review by johnwcallahan — COMMENTED** [review]
Thanks for the contribution @Juliawna!

> Also I'm not sure that I like blue link text inside a red error message.

In the Clone Configs/Disks screen we keep the link text red but underline it:

Screen Shot 2020-09-28 at 9 22 12 AM

Not sure which I like better; tbh the blue link doesn't bother me.

Unrelated, in the non-CMR version there's an invalid DOM nesting warning:

Screen Shot 2020-09-28 at 9 23 50 AM

maybe because `<DialogContentText/>` renders a `<p/>`? Might have to do some conditional stuff there.

**Review by tiffwong — APPROVED** [review]
Great job!

**Pull request merged**